examples/scale/main.rs

```diff
@@ -0,0 +1,108 @@
+use gtk::prelude::*;
+use gtk::{  Adjustment, 
+            Application, 
+            ApplicationWindow, 
+            Grid, 
+            Label, 
+            Orientation, 
+            Scale
+        };
```

Comment:
I definitely don't like importing everything here. In general, I would do that unless the type/function/macro is used more than once.